### ISSUE TITLE
Update docker hub to latest

### ISSUE THREAD

**Issue by SamuelBlickle** [calcom/docker#98]
Could somebody kindly update the latest build on docker hub?
I think there have been many important changes on the codebase within the last 2 months.

Thank you very much!

**Comment by PeerRich** [comment]
@krumware can you help out?

**Comment by PeerRich** [comment]
I do agree @SamuelBlickle we've been a bit behind with the docker maintenance. we hope to soon offer official support for a docker image, right now it's community-ran

**Comment by brunoamaral** [comment]
Tried building from docker and I'm getting an error `error Couldn't find package "@calcom/app-store@*" required by "@calcom/web@1.5.2" on the "npm" registry.`

Guess that's related to the changes?

**Comment by BahrouniYousra** [comment]
@brunoamaral same pb here, have you find a solution plz?

**Comment by brunoamaral** [comment]
@BahrouniYousra , no, I was hoping it would be related with this issue and didn't investigate further.

**Comment by krumware** [comment]
please retry with latest, thanks!

**Comment by krumware** [comment]
will close after resolving most recent build issue
